src/Components/Login/LoginContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {getloginUserThunkCreator} from "../../redux/auth-reducer";
+import Login from "./Login";
+import {connect} from "react-redux";
+
+
+class LoginContainer extends React.Component {
+
+    componentDidMount() {
+        this.props.getloginUserThunkCreator();
```

Comment:
I don't think you want to call it without `dispatch`, without arguments and on `componentDidMount`